server/src/main/java/com/projects/pms/repo/EarningsRepo.java

```diff
@@ -0,0 +1,9 @@
+package com.projects.pms.repo;
+
+import com.projects.pms.models.Earnings;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface EarningsRepo extends JpaRepository<Earnings, Integer> {
```

Comment:
@SunilSunny698 Revisit all the classes and have proper and full names ( not shortcuts for any classes, variables, attributes )